pkg/netpol/connlist/connlist.go

```diff
@@ -204,17 +206,57 @@ func (ca *ConnlistAnalyzer) ConnlistFromK8sCluster(clientset *kubernetes.Clients
 	return ca.getConnectionsList(pe)
 }
 
-// ConnectionsListToString returns a string of connections from list of Peer2PeerConnection objects
-func (ca *ConnlistAnalyzer) ConnectionsListToString(conns []Peer2PeerConnection) string {
+// ConnectionsListToString returns a string of connections from list of Peer2PeerConnection objects in the required output format
+func (ca *ConnlistAnalyzer) ConnectionsListToString(conns []Peer2PeerConnection, output string) (string, error) {
```

Comment:
Pass a formatter (function or interface) instead of the format name as parameter?

Reply:
done - created a new interface - `ConnsFormatte`r , implemented by `TxtFormatter` and `JSONFormatter` and writes the output based on the output arg